tests/e2e/cucumber.hooks.js

```diff
@@ -0,0 +1,31 @@
+const { Before, BeforeAll, AfterAll, After, setDefaultTimeout } = require('@cucumber/cucumber')
+const { firefox } = require('playwright')
+const { deleteFiles } = require('./utils/helpers')
+// default timeout (in milliseconds)
+setDefaultTimeout(100000)
+
+// launch the browser
+BeforeAll(async function () {
+  global.browser = await firefox.launch({
```

Comment:
Let's use chrome browser because it has the largest user base
```suggestion
  global.browser = await chrome.launch({
```

Reply:
[Chrome and ms-edge issue with playwright](https://playwright.dev/docs/browsers#google-chrome--microsoft-edge). Also chrome is not available as direct installation in playwright. Shall we use chromium?

Reply:
yeah please chromium